acoustid/data/source.py

```
@@ -22,32 +23,63 @@ def get_source(conn: AppDB, id: int) -> dict[str, Any] | None:
     return dict(source._mapping) if source is not None else None
 
 
-def find_or_insert_source(
+def find_source_id(
     conn: AppDB, application_id: int, account_id: int, version: str | None = None
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)